src/Client/Updater.php

```diff
@@ -23,6 +23,8 @@
 class Updater
 {
 
+    const MAX_ROOT_DOWNLOADS = 10000000;
```

Comment:
Might be a bit excessive 🙂

Reply:
yep. bumped it down to 1024

Reply:
No problem 🙂 BTW, as an aside, your team and yourself might like to join us to discuss #tuf on the [CNCF Slack](https://cloud-native.slack.com/)! Let me know if you need an invite, just email `trishank.kuppusamy [that-symbol] datadoghq.com`

Reply:
@trishankatdatadog Thanks I just emailed you